week-1-practise/level-1/6-logic-error-3.js

```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trim(word) {
-  return "word".trim();
+  return word.trim();
```

Comment:
What is `word` in this instance then? :)

Reply:
"code your future"